src/components/TheSidebar.vue

```diff
@@ -9,7 +9,7 @@
       </p>
     </div>
 
-    <SmallBoard :listitems="recent">
+    <SmallBoard :listitems="recentArticles">
```

Comment:
Oops, sorry I think I've made a mistake while rebase-ing. The "recent" is correct.